django/core/management/helper.py

```diff
@@ -0,0 +1,66 @@
+from django import setup
+from django.apps import apps
+from django.utils.module_loading import import_string
+
+DJANGO_DEFAULT_IMPORTS = [
+    "from django.core.cache import cache",
+    "from django.conf import settings",
+    "from django.contrib.auth import get_user_model",
+    "from django.db import transaction",
+    "from django.db.models import Avg, Case, Count, F, Max, Min, Prefetch, Q, Sum, When",
+    "from django.utils import timezone",
+    "from django.urls import reverse",
+    "from django.db.models import Exists, OuterRef, Subquery",
+]
+
+CHANGED_IMPORTS = [
+    {"cache": "django.core.cache.cache"},
+    {"settings": "django.conf.settings"},
+    {"get_user_model": "django.contrib.auth.get_user_model"},
+    {"transaction": "django.db.transaction"},
+    {"Avg": "django.db.models.Avg"},
+    {"Case": "django.db.models.Case"},
+    {"Count": "django.db.models.Count"},
+    {"F": "django.db.models.F"},
+    {"Max": "django.db.models.Max"},
+    {"Min": "django.db.models.Min"},
+    {"Prefetch": "django.db.models.Prefetch"},
+    {"Q": "django.db.models.Q"},
+    {"Sum": "django.db.models.Sum"},
+    {"When": "django.db.models.When"},
+    {"timezone": "django.utils.timezone"},
+    {"reverse": "django.urls.reverse"},
+    {"Exists": "django.db.models.Exists"},
+]
```

Comment:
I am not sure that we should add any non-model default imports. Yes, it’s tempting to add a few oft-used things, but preferences will vary between users and projects. I would prefer that we document how users can extend the `shell` function to add extra objects.

Reply:
Yeah I am planing to add functionality that user set there default imports in settings.py:
SHELL_DEFAULT_IMPORT = [
'from django.contrib.auth.models import get_user_model'
]
it they don't set any default imports the shell will only import models of installed applications.